src/infer_request.cc

```diff
@@ -154,17 +154,17 @@ InferenceRequest::TraceInputTensors(
           0, &buffer, &buffer_size, &src_memory_type, &src_memory_type_id);
       if (!status.IsOk()) {
         LOG_STATUS_ERROR(
-            status,
-            std::string(TRITONSERVER_InferenceTraceActivityString(activity)) +
-                ": " + msg + ": tensor: " + name +
-                ": fail to get data buffer: " + status.Message());
+            status, IdString() +
+                        TRITONSERVER_InferenceTraceActivityString(activity) +
+                        ": " + msg + ": tensor: " + name +
+                        ": fail to get data buffer: " + status.Message());
         return status;
       }
 
       if (buffer_size != byte_size) {
         LOG_STATUS_ERROR(
             status,
-            std::string(TRITONSERVER_InferenceTraceActivityString(activity)) +
+            IdString() + TRITONSERVER_InferenceTraceActivityString(activity) +
```

Comment:
`IdString()` is too specific, if we are going to log more request info, the name will need to be changed. Can be something like `LogRequest()`